.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -7,25 +7,25 @@ assignees: ''
 
 ---
 
-**Expected behavior**
+## Expected behavior
 A clear and concise description of what you expected to happen.
 
-**Describe the bug**
+## Describe the bug
 A clear and concise description of what the bug is.
 
-**To Reproduce**
+## To Reproduce
 Steps to reproduce the behavior:
 1. 
 2. 
 3. 
 4. 
 
-**Screenshots**
-If applicable, add screenshots to help explain your problem.
+## Screenshots/output
+If applicable, add screenshots or program output to help explain your problem.
```

Comment:
Maybe we should reorder this to put screenshots last? It's much easier to search by word with formatted quote text rather than screenshots? 

```suggestion
If applicable, add program output with triple backticks, a gist of your output, or a screenshot to help explain your problem.
```

Reply:
This is an excellent clarification.